kong-ngx-build

```diff
@@ -387,6 +407,83 @@ main() {
   succ "Build finished in $SECONDS seconds. Enjoy!"
 }
 
+parse_version() {
+  [[ -z $1 ]] && fatal 'missing arg $1 when invoking parse_version()'
+  [[ -z $2 ]] && fatal 'missing arg $2 when invoking parse_version()'
+
+  local subj=$1
+  local -n ver=$2
```

Comment:
Note `local -n` only works on bash >= 5, which is not that new, but for instance, os x comes bundled with 3.2

Reply:
Worst case, we can revert this commit and go back to the previous implementation, I made sure to not override your first patch.